dask/dataframe/tests/test_multi.py

```diff
@@ -847,6 +847,26 @@ def test_merge(how, shuffle):
     #         pd.merge(A, B, left_index=True, right_on='y'))
 
 
+def test_merge_how_raises():
+    left = pd.DataFrame(
+        {
+            "A": ["A0", "A1", "A2", "A3"],
+            "B": ["B0", "B1", "B2", "B3"],
+        }
+    )
+    right = pd.DataFrame(
+        {
+            "A": ["C0", "C1", "C2", "C3"],
+            "B": ["D0", "D1", "D2", "D3"],
+        }
+    )
+
+    with pytest.raises(
+        Exception, match="dask.dataframe.merge does not support how='cross'"
```

Comment:
We'll need this corresponding change too

```suggestion
        ValueError, match="dask.dataframe.merge does not support how='cross'"
```